### PR TITLE
Gotoplanb/12/open source licenses.

### LINKED ISSUE #12: Open Source Licenses 101
Introduce the world to the kinds of licenses that go into open source libraries, and how to interpret them when using said libraries in projects, and what that means for the project source code.

### PR DESCRIPTION
**Pull request by gotoplanb**
Closes #12

### CODE REVIEW

File: _posts/2015-02-02-open-source-licenses-101.markdown [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Simply an open-source license must comply with the criteria of the [Open Source Initative](http://opensource.org/osd), which is a [non-profit 501(c)3 registered in California](http://opensource.org/about). The critera (and short definitions) are:
+
+1. No fees for redistribution
+2. Must provide source code
+3. Allows for derived works
+4. Integrity of the author's source instead of so much modification original is unrecognizable
+5. No discrimation
+6. No restrictions on why the source is used
+7. License details are included with the source
+8. Source must me distributable by itself
```
Comment (tikipatel) [inline _posts/2015-02-02-open-source-licenses-101.markdown]: Source must be*(?)

File: _posts/2015-02-02-open-source-licenses-101.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+> You may create and distribute a Larger Work under terms of Your choice, provided that You also comply with the requirements of this License for the Covered Software. If the Larger Work is a combination of Covered Software with a work governed by one or more Secondary Licenses, and the Covered Software is not Incompatible With Secondary Licenses, this License permits You to additionally distribute such Covered Software under the terms of such Secondary License(s), so that the recipient of the Larger Work may, at their option, further distribute the Covered Software under the terms of either this License or such Secondary License(s).
+
+You can modify MPL works in your larger work without the MPL license having to bubble up to the rest of the larger work like in GPLv3.
+
+Also worth noting that if you break some rule of this license, you can fix your violations and then be back in the good graces of the licensor. 
+
+### More licenses
+
+There are many more open source licenses. Here's a [list of other popular licenses](http://opensource.org/licenses/).
+
```
Comment (tikipatel) [inline _posts/2015-02-02-open-source-licenses-101.markdown]: Perhaps a link to http://choosealicense.com/licenses/ and http://choosealicense.com/ (?)

**Comment by gotoplanb** [comment]
@ustic I made #21 to think and write about attribution at some point.

**Comment by gotoplanb** [comment]
@tikipatel Thanks for the typo catch and suggestion of license wizard link. Added.

**Pull request merged**